spec/tasks/one_off_spec.rb

```diff
@@ -125,6 +125,32 @@
         end
       end
 
+      context "with many ignored versions" do
+        before do
+          50.times do |i|
```

Comment:
🎱 Should be able to do a `create_list(:version, 50, **opts)` but may need to update the factory to use sequences for that to work, same with the below one.